app/src/main/java/com/reminder/hydration/EntryViewModel.kt

```diff
@@ -0,0 +1,98 @@
+package com.reminder.hydration
+
+import android.app.Notification
+import android.util.Log
+import android.widget.RadioButton
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+
+class EntryViewModel(private val sharedPrefManager: SharedPrefManager) : ViewModel() {
+
+    private val wakeUpTimeData: MutableLiveData<Int> = MutableLiveData()
+    private val sleepTimeData: MutableLiveData<Int> = MutableLiveData()
+    private val drunkenWaterAmountData: MutableLiveData<Int> = MutableLiveData()
+    private val soundTypeData: MutableLiveData<Int> = MutableLiveData()
+    private val notifTypeData: MutableLiveData<Int> = MutableLiveData()
+    private val warning: MutableLiveData<String> = MutableLiveData<String>()
+    private val error: MutableLiveData<String> = MutableLiveData<String>()
+
+    fun wakeUpTime(): LiveData<Int> = wakeUpTimeData
+    fun sleepTime(): LiveData<Int> = sleepTimeData
+    fun drunkenWaterAmount(): LiveData<Int> = drunkenWaterAmountData
+    fun notifType(): LiveData<Int> = notifTypeData
+    fun soundType(): LiveData<Int> = soundTypeData
+
+
+    fun loadData() {
+        wakeUpTimeData.value = sharedPrefManager.load(wakeUpTimeKey, 0)
+        sleepTimeData.value = sharedPrefManager.load(sleepTimeKey, 0)
+        drunkenWaterAmountData.value = sharedPrefManager.load(waterNumKey, 0)
+        notifTypeData.value = sharedPrefManager.load(notifTypeKey, NotifType.NOTIFICATION.value)
+        soundTypeData.value = sharedPrefManager.load(soundTypeKey, SoundType.SOUND.value)
+    }
```

Comment:
This function is going to create an instance of that data class and send it to one livedata you have.